src/pages/ProcessJoinTeam.vue

```diff
@@ -138,7 +162,7 @@ export default {
 					mutation: joinTeam,
 					variables: {
 						team_id: this.teamId,
-						team_recruitment_id: this.teamRecruitmentId,
+						team_recruitment_id: this.recruitmentIdMutationVariable,
```

Comment:
Do we need to check if `recruitmentIdMutationVariable` ends up being `null` before sending of this mutation, or are we OK with it being `null`?

Reply:
It works, but no harm in making both of these conditional